src/routes/event/[username]/[eventId]/qr/+page.svelte

```diff
@@ -1,12 +1,24 @@
 <script lang="ts">
+	import { page } from '$app/stores';
 	import Blur from '$lib/components/Blur.svelte';
 	import EventQrCard from '$lib/components/EventQRCard.svelte';
+	import generateQRCode from '$lib/utilities/generateQRCode';
+	import { onMount } from 'svelte';
+
+	export let data;
+
+	let qrCodeDataUrl: string;
+
+	onMount(async () => {
+		let qr = `http://localhost:5173/event/${$page.params.username}/${data.overview.eventId}`;
```

Comment:
Hardcoding `http://localhost:5173` will give us errors in the future..
We can get the real URL of the project from the `$page` store